recipes/premake/5.x/conanfile.py

```diff
@@ -49,58 +46,26 @@ def package_id(self):
         del self.info.settings.compiler
 
     def requirements(self):
-        if self.settings.os != "Windows":
-            self.requires("util-linux-libuuid/2.39")
+        self.requires("libcurl/[>=7.78.0 <9]")
```

Comment:
Version ranges are currently only allowed for a handful of recipes. Is libcurl one of them?

Reply:
Yes.